src/Cimpress.Nancy.Logging/LoggingBootstrapperExtender.cs

```diff
@@ -89,7 +89,7 @@ public Response OnBeforeRequest(NancyContext context, IDictionary<string, object
             var bodyString = new StreamReader(request.Body).ReadToEnd();
             request.Body.Position = 0;
 
-            var isBodyJson = string.Equals(request.Headers.ContentType, "application/json", StringComparison.OrdinalIgnoreCase);
+            var isBodyJson = string.Equals(request.Headers.ContentType ?? string.Empty, "application/json", StringComparison.OrdinalIgnoreCase);
```

Comment:
Perhaps, but it wasn't being set when I was testing it.

We have no control over how the Content-Type header is set. That's based upon Nancy and the clients connecting to the service.

This is in the logging project - it shouldn't have the power to dictate what is required or not. The less we have to require, the better. This check is simply for logging the body as either a string or a JSON string. If the content type is not provided, then we won't store it as JSON. I don't see any harm there and I don't see how it's not robust.